jax/core.py

```diff
@@ -57,7 +57,13 @@ def skipping_checks():
 # -------------------- jaxprs --------------------
 
 class Jaxpr(object):
-  def __init__(self, constvars, invars, outvars, eqns):
+  constvars: List['Var']
```

Comment:
I wonder if we should be standardizing on Tuple as the sequence of choice, unless we need to support mutation. Tuples are also faster. (but the typing hints are annoying: Tuple['Var', ...])

Reply:
I have no objection, although that should happen in another change (this particular annotation codifies the world as it is.)

Reply:
Yeah could do! One thing I've noticed is that list comprehensions are nicer to write than `tuple(generator_expression)`, which is one way lists are nicer to work with.